diff --git a/configure b/configure
index 5da78e9..bc803a8 100755
--- a/configure
+++ b/configure
@@ -3771,24 +3771,6 @@ printf "%s\n" "no" >&6; }
 fi
 
 
-case $PERL in
- '')
-   as_fn_error $? "perl not found" "$LINENO" 5
-   ;;
- *' '* | *'	'*)
-  { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: The path to your Perl contains spaces or tabs.
-This will likely cause build failures later or unusable programs.
-Unless you're specifying a full string for a #! line,
-as in \"/usr/bin/env perl\",
-please use a path without spaces and try again." >&5
-printf "%s\n" "$as_me: WARNING: The path to your Perl contains spaces or tabs.
-This will likely cause build failures later or unusable programs.
-Unless you're specifying a full string for a #! line,
-as in \"/usr/bin/env perl\",
-please use a path without spaces and try again." >&2;}
-  ;;
-esac
-
 # Save details about the selected perl interpreter in config.log.
 { echo "$as_me:$LINENO: $PERL --version" >&5
    ($PERL --version) >&5 2>&5
