diff --git a/lib/am/distdir.am b/lib/am/distdir.am
index 4797492..bb9017b 100644
--- a/lib/am/distdir.am
+++ b/lib/am/distdir.am
@@ -442,6 +442,8 @@ distcheck: dist
 ## Make the new source tree read-only.  Distributions ought to work in
 ## this case.  However, make the top-level directory writable so we
 ## can make our new subdirs.
+## On z/OS, tag all the new files as ASCII
+	chtag -R -tcISO8859-1 $(distdir)/*
 	chmod -R a-w $(distdir)
 	chmod u+w $(distdir)
 	mkdir $(distdir)/_build $(distdir)/_build/sub $(distdir)/_inst
