diff --git a/lib/am/distdir.am b/lib/am/distdir.am
index 6ba8dff..79e2341 100644
--- a/lib/am/distdir.am
+++ b/lib/am/distdir.am
@@ -537,6 +537,7 @@ distcheck: dist
 ## Make the new source tree read-only.  Distributions ought to work in
 ## this case.  However, make the top-level directory writable so we
 ## can make our new subdirs.
+	chtag -R -tcISO8859-1 $(distdir)/*
 	chmod -R a-w $(distdir)
 	chmod u+w $(distdir)
 	mkdir $(distdir)/_build $(distdir)/_build/sub $(distdir)/_inst
