diff --git a/Makefile.in b/Makefile.in
index 8e80b76..f201759 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -3299,6 +3299,10 @@ distcheck: dist
 	  unzip $(distdir).zip ;;\
 	esac
 	chmod -R a-w $(distdir)
+#
+# On z/OS, tag all the untar'ed files as ASCII
+#
+#chtag -R -tcISO8859-1 $(distdir)
 	chmod u+w $(distdir)
 	mkdir $(distdir)/_build $(distdir)/_build/sub $(distdir)/_inst
 	chmod a-w $(distdir)
